lib/tapioca/generator.rb

```diff
@@ -510,14 +510,13 @@ def compile_gem_rbi(gem)
     def compile_dsl_rbi(constant, contents, outpath: config.outpath)
       return if contents.nil?
 
-      command = format(config.generate_command, constant.name)
       constant_name = Module.instance_method(:name).bind(constant).call
       rbi_name = constant_name.underscore + ".rbi"
       filename = outpath / rbi_name
 
       out = String.new
       out << rbi_header(
-        command,
+        "tapioca dsl",
```

Comment:
Let's make this:
```suggestion
        "tapioca dsl #{constant_name}",
```
since it should always be safe to run the DSL command for a single file and it will always be faster than a full run.